mkdocs.yml

```diff
@@ -12,6 +12,7 @@ nav:
       - Getting Started Guide: getting-started/getting-started.md
       - Installation: getting-started/installation.md
       - In-Cluster Operator: getting-started/in-cluster-operator.md
+      - Serve mode: getting-started/serve-mode.md
```

Comment:
What's the reason it is in the getting started section? Might be better in reference? 
maybe under https://docs.k8sgpt.ai/reference/cli/ or creating a new section 
-- I would assume that people who are getting started would not use this option first?

Reply:
You're right @AnaisUrlichs . I'll change